src/ensembl/src/shared/components/checkbox/Checkbox.scss

```diff
@@ -31,6 +30,7 @@
 }
 
 .defaultLabel {
+  cursor: pointer;
```

Comment:
Do we need something like this to not show the pointer over the label when it is disabled:
```
defaultCheckbox  + label{
    cursor: pointer;
}

.defaultCheckbox.disabled  + label{
    cursor: normal;
}

```

Reply:
👍 

I also updated the Checkbox stories to reflect different behaviour of the label for default and disabled checkboxes.